spec/acceptance/discovery_spec.rb

```diff
@@ -1,13 +1,58 @@
 require 'spec_helper_acceptance'
 
 describe 'Scenario: install foreman-proxy with discovery plugin'  do
+  root = case host_inventory['facter']['os']['name']
+         when 'Debian'
+           '/srv/tftp'
+         when 'Ubuntu'
+           if host_inventory['facter']['os']['release']['major'].to_f >= 20.04
+             '/srv/tftp'
+           else
+             '/var/lib/tftpboot'
+           end
+         else
+           '/var/lib/tftpboot'
+         end
+
   before(:context) { purge_foreman_proxy }
 
-  include_examples 'the example', 'discovery.pp'
+  context 'without params' do
+    include_examples 'the example', 'discovery.pp'
+
+    it_behaves_like 'the default foreman proxy application'
+
+    describe command('curl -sk https://127.0.0.1:8443/features | grep -q discovery') do
+      its(:exit_status) { should eq 0 }
```

Comment:
I realize it was this way before, but I think it'd be better to check the output in Ruby. That way if it fails, you see the real output and can diff rather than a non-zero exit code.
```suggestion
    describe command('curl -sk https://127.0.0.1:8443/features') do
      its(:stdout) { should contain('discovery') }
      its(:exit_status) { should eq 0 }
```

Reply:
Done.